diff --git a/dist/plugins/markdown-shortcut/index.js b/dist/plugins/markdown-shortcut/index.js
--- a/dist/plugins/markdown-shortcut/index.js
+++ b/dist/plugins/markdown-shortcut/index.js
@@ -97,7 +97,7 @@ function pickTransformersForActivePlugins(pluginIds, allowedHeadingLevels) {
       ...CODE,
       dependencies: [CodeBlockNode],
       replace: (parentNode, _children, match) => {
-        const codeBlockNode = $createCodeBlockNode({ code: "", language: match[1] ?? "", meta: "" });
+        const codeBlockNode = $createCodeBlockNode({ code: "", language: match[2] ?? "", meta: "" });
         parentNode.selectPrevious();
         parentNode.replace(codeBlockNode);
         setTimeout(() => {
