diff --git a/src/pg_cron.c b/src/pg_cron.c
index e0ca973..4d51b2c 100644
--- a/src/pg_cron.c
+++ b/src/pg_cron.c
@@ -14,6 +14,8 @@
 #include <sys/resource.h>
 
 #include "postgres.h"
+#include "commands/async.h"
+#include "miscadmin.h"
 #include "fmgr.h"
 
 /* these are always necessary for a bgworker */
@@ -1908,7 +1910,7 @@ CronBackgroundWorker(Datum main_arg)
 	/* Post-execution cleanup. */
 	disable_timeout(STATEMENT_TIMEOUT, false);
 	CommitTransactionCommand();
-	ProcessCompletedNotifies();
+	/* ProcessCompletedNotifies removed */
 	pgstat_report_activity(STATE_IDLE, command);
 	pgstat_report_stat(true);
 
@@ -2025,7 +2027,7 @@ ExecuteSqlString(const char *sql)
 		 */
 		oldcontext = MemoryContextSwitchTo(parsecontext);
 		#if PG_VERSION_NUM >= 100000
-			querytree_list = pg_analyze_and_rewrite(parsetree, sql, NULL, 0,NULL);
+			querytree_list = pg_analyze_and_rewrite_fixedparams(parsetree, sql, NULL, 0, NULL);
 		#else
 			querytree_list = pg_analyze_and_rewrite(parsetree, sql, NULL, 0);
 		#endif