--- a/src/port/path.c
+++ b/src/port/path.c
@@ -714,7 +714,11 @@
 void
 get_lib_path(const char *my_exec_path, char *ret_path)
 {
-	make_relative_path(ret_path, LIBDIR, PGBINDIR, my_exec_path);
+	char const * const nix_pglibdir = getenv("NIX_PGLIBDIR");
+	if(nix_pglibdir == NULL)
+		make_relative_path(ret_path, LIBDIR, PGBINDIR, my_exec_path);
+	else
+		make_relative_path(ret_path, nix_pglibdir, PGBINDIR, my_exec_path);
 }
 
 /*
@@ -723,7 +727,11 @@
 void
 get_pkglib_path(const char *my_exec_path, char *ret_path)
 {
-	make_relative_path(ret_path, PKGLIBDIR, PGBINDIR, my_exec_path);
+	char const * const nix_pglibdir = getenv("NIX_PGLIBDIR");
+	if(nix_pglibdir == NULL)
+		make_relative_path(ret_path, PKGLIBDIR, PGBINDIR, my_exec_path);
+	else
+		make_relative_path(ret_path, nix_pglibdir, PGBINDIR, my_exec_path);
 }
 
 /*
