diff --git a/node_modules/cross-fetch/dist/browser-ponyfill.js b/node_modules/cross-fetch/dist/browser-ponyfill.js
index 7a795ac..c01e55f 100644
--- a/node_modules/cross-fetch/dist/browser-ponyfill.js
+++ b/node_modules/cross-fetch/dist/browser-ponyfill.js
@@ -544,3 +544,4 @@ exports.Headers = __self__.Headers
 exports.Request = __self__.Request
 exports.Response = __self__.Response
 module.exports = exports
+module.exports = fetch
\ No newline at end of file
